README.md

```diff
@@ -51,12 +53,12 @@ Typical build configurations can be easily summoned as detailed in
 
 Afterwards enter you build directory and fire the build process:
 
-        cd build_rel
+        cd builddir
```

Comment:
Same here

Reply:
Well, this change is (clearly?) wrong, because the release build process shall be initiated, and some lines above the matching example uses the directory `build_rel`, and not `builddir`, so this becomes disconnected.

I, personally, hate `builddir`, and maybe how VScode handles this. I always have multiple build dirs set up; at least one for GCC, one for clang, one for release.

Reply:
If we want to explain how to make a release build, I guess we should say so. I'll adapt the text accordingly. And since Fini hates `builddir`, I'll revert that to build.